panacea/query_client.go

```diff
@@ -72,6 +73,14 @@ func NewQueryClient(ctx context.Context, chainID, rpcAddr string, trustedBlockHe
 	if err != nil {
 		return nil, err
 	}
+	// call refresh every minute
+	go func() {
+		for {
+			time.Sleep(1 * time.Minute)
+			err = refresh(ctx, lc, trustOptions.Period)
+			logrus.Info("light client refresh error: ", err)
```

Comment:
I think that error log will occur even if no error occurs.
So how about modifying it like this:
```suggestion
			if err := refresh(ctx, lc, trustOptions.Period); err != nil {
			    logrus.Info("light client refresh error: ", err)
			}
```

Reply:
And we have to think about the behavior when refresh fails.
As for now, whether to output only the log or terminate the process.

Reply:
Ah, I got it. Thanks! I reflect it.